test/tests/functional/pbs_multi_sched.py

```diff
@@ -1370,3 +1370,29 @@ def test_multisched_sched_cycle_length(self):
               " sched_cycle_length of 2 seconds"
         self.scheds['sc2'].log_match(msg, starttime=t)
         self.server.cleanup_jobs()
+
+    def test_job_sort_formula_threshold(self):
+        """
+        Test the scheduler attribute job_sort_formula_threshold for multisched
+        """
+        # Multisched setup
+        self.setup_sc3()
+        self.setup_queues_nodes()
```

Comment:
setup_queues_nodes() is creating a set of nodes and queues some of which are not required for this test case. May be you can just create only the required nodes/queues for this test case. This is just a suggestion.  

Thanks for making all the code changes. Except the above all the code changes look good to me.

Reply:
@suresh-thelkar I have addressed your suggestion. Thanks

Reply:
@Bhagat-Rajput, Thanks for making the changes. I sign off.